### ISSUE TITLE
panic: send on closed channel (rel3 branch)

### ISSUE THREAD

**Issue by aleks-m** [rwynn/monstache#321]
Hi. We are running monstache 3.26.1 in docker.

And frequently see the following error at monstache shutdown:

```
panic: send on closed channel

goroutine 501 [running]:
gopkg.in/olivere/elastic%2ev5.(*BulkProcessor).Add(...)
/go/pkg/mod/gopkg.in/olivere/elastic.v5@v5.0.76/bulk_processor.go:374
main.doIndexing(0xc0000e9080, 0xc000136540, 0xc0001b45b0, 0xc00036eea0, 0xc00a830e40, 0xc000092540, 0xc0022edf98)
/app/monstache.go:2845 +0x1173
main.doIndex(0xc0000e9080, 0xc000136540, 0xc0001b45b0, 0xc00036eea0, 0xc00a830e40, 0x0, 0x0)
/app/monstache.go:2903 +0x10d
main.main.func6(0xc000125ba0, 0xc000306540, 0xc0000e9080, 0xc000136540, 0xc0001b45b0, 0xc00036eea0)
/app/monstache.go:4220 +0xc4
created by main.main
/app/monstache.go:4217 +0x1309
```

Here is our config:
```
              workers = ["worker0", "worker1", "worker2", "worker3"]
              gzip = false
              stats = true
              index-stats = false
              elasticsearch-max-conns = 6
              elasticsearch-max-bytes = 103809024 # 99Mb
              elasticsearch-retry = true
              dropped-collections = false
              dropped-databases = false
              replay = false
              resume = true
              resume-write-unsafe = true
              namespace-exclude-regex = ''
              verbose = false
              exit-after-direct-reads = false
              prune-invalid-json = true
              enable-http-server = true

              [gtm-settings]
              channel-size = 512
```

**Comment by rwynn** [comment]
Hi @aleks-m I pushed a [commit](https://github.com/rwynn/monstache/commit/8f458bda101c608127819095eb80bfdab2471e98) that hopefully will fix the shutdown.  If you have any feedback on whether or not that works I would appreciate it.  I will push a new docker image when I get some time unless you notice some issues.

**Comment by rwynn** [comment]
@aleks-m this should be fixed now in `v3.26.3`.
